electron/utils.ts

```diff
@@ -549,6 +557,47 @@ function quoteIfNecessary(stringToQuote: string) {
   return stringToQuote
 }
 
+/**
+ * Checks if we're (1) on a Steam Deck and (2) running in Gaming Mode
+ */
+function isSteamDeckInGamingMode(): boolean {
+  if (!isLinux) {
+    return false
+  }
+
+  // Credits for this implementation go to https://gitlab.com/parallel-launcher/parallel-launcher
+
+  // Check if we're on a Deck by reading out /etc/os-release
+  let osReleasePath = '/etc/os-release'
+  // Just in case someone is *not* running the Flatpak
+  if (isFlatpak) {
+    osReleasePath = '/run/host' + osReleasePath
+  }
+
+  let foundDeckVariant = false
+  for (const line of readFileSync(osReleasePath, 'utf-8').split('\n')) {
+    const [key, value] = `${line}`.split('=')
+    if (key === 'VARIANT_ID') {
+      foundDeckVariant = value === 'steamdeck'
+      break
+    }
+  }
```

Comment:
Not 100% if syntax correct, but i think this is cleaner.

```suggestion
  const fileContent = readFileSync(osReleasePath).toString() // 'utf-8' is default
  const foundDeckVariant = fileContent.split('\n').find((line: string) => {
    const [key, value] = line.split('=')
    return key === 'VARIANT_ID' && value ==='steamdeck'
  }
```

Reply:
Each day I learn a new thing about JS